Directory.Build.props

```diff
@@ -30,15 +31,22 @@
         <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\CustomAnalysisRules.ruleset</CodeAnalysisRuleSet>
       </PropertyGroup>
     </Otherwise>
-  </Choose>
+   </Choose>
 
-  <ItemGroup>
+   <ItemGroup>
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)\stylecop.json" Link="stylecop.json" />
-  </ItemGroup>
+   </ItemGroup>
 
-  <ItemGroup>
+   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-19270-01" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" />
-  </ItemGroup>
+   </ItemGroup>
 
+   <Choose>
+    <When Condition="$(MSBuildProjectName)!='SmartLauncher' AND $(MSBuildProjectName)!='Microsoft.Health.Extensions.BuildTimeCodeGenerator' AND !$(MSBuildProjectName.Contains('Test'))">
```

Comment:
SmartLauncher and BuildTimeCodeGenerator need good refactoring to enable styleCop
Glimpse of BuildTimeCodeGenerator errors -
![image](https://user-images.githubusercontent.com/57157506/71605456-a9245d00-2b1d-11ea-9f99-0d30613a4795.png)

Reply:
We probably want to create issues to track this if it's something we want to do.

Reply:
sure

Reply:
Created Issue :

ID | Title | Assigned To | State | Area Path | Tags | Comments
-- | -- | -- | -- | -- | -- | --
71779 | Enable FxCopAnalyzers for SmartLauncher and BuildTimeCodeGenerator projects | Richa Bansal | Active | Health\Patient Engagement\Resolute |   | 0